src/masternodes/govvariables/attributes.h

```diff
@@ -85,6 +86,9 @@ enum DFIPKeys : uint8_t {
     GovernanceEnabled    = 'q',
     CFPPayout            = 'r',
     EmissionUnusedFund   = 's',
+    Limit                = 't',
```

Comment:
@Mixa84  Is it mandatory that one key is only used for one DFIP? I kept it "generic" so this Limit could be reused if anyother DFIP also needs a limit, same for Token and WithdrawHeight. Verifications are done in a way that it can be used for other DFIPs too. and within the params its anyway in the path of the DFIP.

Reply:
I thought about that and I would like to reuse them for other DFIPs, but I think it is not clear from the text what it does.
`v0/params/dfip2211d/limit/1`, `v0/params/dfip2211d/token/1`
@prasannavl what do you say on this?